structure/string.go

```diff
@@ -33,7 +34,12 @@ func (t *TxStructure) Set(key []byte, value []byte) error {
 // Get gets the string value of a key.
 func (t *TxStructure) Get(key []byte) ([]byte, error) {
 	ek := t.encodeStringDataKey(key)
-	value, err := t.reader.Get(context.TODO(), ek)
+	ctx := context.TODO()
+	isMeta := bytes.Equal([]byte("SchemaVersionKey"), key)
```

Comment:
There are loadSchemeDiff and loadFull, your mean just the version key?

Reply:
Yes. Because when there are diff load or full load, they will printed with "load infoSchema", but they didn't.
Also, I didn't see any ddl jobs in `admin show ddl jobs`.